pkg/fqdn/regexpmap/regexp_map_test.go

```diff
@@ -54,15 +85,16 @@ func (ds *RegexpMapTestSuite) TestREMapInsertLookup(c *C) {
 
 	// Does LookupValue match Added values internally?
 	match := m.LookupContainsValue("foo.bar.com.", "ID1")
+
 	c.Assert(match, Equals, true, Commentf("No match for key & value that should match"))
 	match = m.LookupContainsValue("foo.bar.com.", "ID2")
 	c.Assert(match, Equals, true, Commentf("No match for key & value that should match"))
 	match = m.LookupContainsValue("foo.bar.com.", "ID3")
 	c.Assert(match, Equals, false, Commentf("Match for key that matches & value that does not match"))
 	match = m.LookupContainsValue("notabar.com.", "ID2")
 	c.Assert(match, Equals, false, Commentf("Match for key that does not matche & value that does match"))
-
 	// Does removing an entry also remove it from lookup returns
+
```

Comment:
was there a reason to move this empty line? the comment is meant to go with the code below.